server/src/controllers/job_posts/getAllJobPosts.js

```diff
@@ -0,0 +1,85 @@
+// Get All Job Posts
+// a) GET request to get all job posts [id, title, employer, application_close_date, state]
+// Return the id, title, employer, application_close_date, state values of ALL job posts
+
+// b) GET request to get all public job posts 
+// Return the id, title, employer, location, application_close_date of job posts where state is "Active"
```

Comment:
Can you also return the job_type for these two? So that it can be displayed on the frontend side like this: https://www.figma.com/design/ckkCZEVkae0kIXUwaQPbib/High-Fidelity-Designs?node-id=3736-1502&node-type=canvas&t=nbe7GK9EivLeGPCV-0